assets/src/blocks/Cookies/CookiesEditor.js

```diff
@@ -1,18 +1,44 @@
 import {CookiesFrontend} from './CookiesFrontend';
 
+const {InspectorControls} = wp.blockEditor;
+const {PanelBody} = wp.components;
+const {__} = wp.i18n;
+
 export const CookiesEditor = ({attributes, isSelected, setAttributes}) => {
   const toAttribute = attributeName => value => {
     if (isSelected) {
       setAttributes({[attributeName]: value});
     }
   };
 
-  return (
+  const renderEdit = () => (
+    <>
+      <InspectorControls>
+        <PanelBody title={__('Learn more about this block ', 'planet4-blocks-backend')} initialOpen={false}>
+          <p className="components-base-control__help">
+            <a target="_blank" href="https://planet4.greenpeace.org/content/blocks/counter/" rel="noreferrer">
```

Comment:
And not the correct link (text) either 😉

Reply:
Fixed!